client/src/components/ExpandingFormset/index.js

```diff
@@ -0,0 +1,43 @@
+import $ from 'jquery';
+
+export class ExpandingFormset {
+  constructor(prefix, opts = {}) {
+    this.opts = opts;
+    const addButton = $('#' + prefix + '-ADD');
```

Comment:
The big gotcha with removing jQuery here is that we're very much reliant on inline `<script>` tags being executed when a new form's HTML is inserted - this is a standard feature of jQuery but not the native DOM methods.

(This is certainly not the first time this has come up - modal-workflow was probably the last thing we contemplated eliminating jQuery from, but didn't, for this reason - so it's probably time we bit the bullet and added a "run all inline scripts in this DOM fragment" helper function for these situations.)

Reply:
True. But we could defer the calling with a static method on this class that waits for Dom ready. Or move the jQuery but to when this gets instantiated. 

However, this may not be worth it as it's something that will just be solved for us without thinking when on Stimulus. No instance will get instantiated until the DOM is ready.

Reply:
I think you might be referring to the inline `<script>` block that does the InlinePanel initialisation? That's not the one I mean - I'm talking about inline scripts that exist within the HTML fragment for the new child form being inserted, e.g. because that form contains a rich text field, chooser widget or similar. For those, the DOM ready event doesn't enter into it, because the HTML insertion is happening well after the page has loaded.

Reply:
ok - will not push on this further - I think we can find a better way to do this but it's probably not worth doing while we know we have a more reliable approach coming with Stimulus.